### DISCUSSION TITLE
@MAXTRANSFERSIZE, @BLOCKSIZE params are not supported when performing backups to URL

### DISCUSSION THREAD

**Discussion by RoD-OlBilyk** [olahallengren/sql-server-maintenance-solution#544] [Questions & Answers]
Hi,

EXECUTE master.dbo.DatabaseBackup @Databases = 'exampleDB', @BackupType = 'LOG' ,
 @Compress = 'Y', @Encrypt = 'Y', @EncryptionAlgorithm = 'AES_128', @ServerCertificate = 'SomeCert', 
 @Verify = 'Y', @LogToTable='Y',
 @URL = 'https://example.blob.core.windows.net/example',  @Credential = 'https://example.blob.core.windows.net/example',
 @MaxTransferSize = 4194304,@Blocksize = 65536

Version: 2020-12-31 18:58:56
Source: https://ola.hallengren.com

generates errs as follows (but in reality, it is unhappy with @Credential parameter (remove it & everything works perfectly). ).

Msg 50000, Level 16, State 4, Procedure master.dbo.DatabaseBackup, Line 2278 [Batch Start Line 4]
The value for the parameter @BlockSize is not supported.
	
Msg 50000, Level 16, State 4, Procedure master.dbo.DatabaseBackup, Line 2278 [Batch Start Line 4]
The value for the parameter @MaxTransferSize is not supported.
	
Msg 50000, Level 16, State 1, Procedure master.dbo.DatabaseBackup, Line 2278 [Batch Start Line 4]
The documentation is available at https://ola.hallengren.com/sql-server-backup.html.

**Comment by olahallengren** [comment]
There are some limitations when you are backing up to page blob.

The solution is to switch to block blob and Shared Access Signature.
https://docs.microsoft.com/en-us/sql/relational-databases/backup-restore/sql-server-backup-to-url?view=sql-server-2017
